### PR TITLE
Refactor the SKILL.md file for the Playwright skill 

### PR DESCRIPTION
**Pull request by tanyafortuna**
# Refactor AC-to-Playwright skill with subagent extraction and progressive disclosure

## Overview

Refactored `accelint-ac-to-playwright` skill from monolithic structure to progressive disclosure pattern with separate mode files and specialized subagent validators.

### Skill Structure
- **SKILL.md** (~100 lines, down from ~500+): Lightweight orchestrator with mode detection and context management
- **assessment-mode.md** (~250 lines): Complete assessment workflow with issue classification and interactive clarification
- **conversion-mode.md** (~165 lines): Full conversion pipeline with validation protocol

### Subagents Extracted (9 total)

**Validation subagents** (parallel-executable):
1. `validate-targets.md` - Target format and controlled vocabulary validation
2. `validate-gherkin-structure.md` - .feature file structure validation
3. `validate-bullet-format.md` - Markdown bullet format validation
4. `validate-preconditions.md` - Given step validation (context setup patterns)
5. `validate-actions.md` - When step validation (verbs, values, coordinates)
6. `validate-assertions.md` - Then step validation (measurability, explicitness)

**Transformation/diagnostic subagents**:
7. `generate-names.md` - Suite/test name generation and slug derivation
8. `diagnose-schema-errors.md` - JSON plan validation failure diagnosis
9. `diagnose-translation-errors.md` - Translation script runtime error diagnosis

### Assessment Mode Improvements

**Issue Classification System**
- Type-based classification (not count-based): PERFECT (zero issues), MIXED (minor issues only), BAD (any major issues)
- **Minor issues** (quick fixes): incomplete targets, invalid vocabulary, missing quotes
- **Major issues** (require discussion): vague verbs, unmeasurable outcomes, missing targets, structural violations

**Interactive Clarification Mode**
- User chooses response format when BAD AC detected:
  - Option (a): Full question list for batch review
  - Option (b): Interactive one-question-at-a-time with incremental updates
- Handles both file paths and pasted AC text
- LLM can update AC incrementally (protects against conversation interruption) or user can update manually

## Benefits

- **Context efficiency**: Load only needed validation logic on-demand
- **Parallel validation**: All 6 assessment validators can run simultaneously
- **Better error handling**: Standardized input/output contracts with line-number precision
- **Improved UX**: Flexible clarification modes based on issue severity and user preference
- **Maintainability**: Separated concerns with clear boundaries between orchestration and validation logic

### CODE REVIEW

**Comment by tanyafortuna** [comment]
@kevanvincent when you have time, could you review this? There are more optimizations I would like to make but I am out of time before my PTO. I will attack them when I'm back, but this should be a good first iteration to try to improve the quality of the agentic part of the skill (and hopefully that will be validated by rerunning the evaluation post-merge of this).

https://github.com/gohypergiant/agent-skills/pull/119 should be merged first

There will likely be merge conflicts with the version/changelog after that is merged, but hopefully they are easy to resolve.

File: skills/accelint-ac-to-playwright/references/conversion-mode.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+1. **Run Assessment mode**:
+  - Run Assessment mode against all input files and report pass/fail result.
+  - If Assessment mode reported any failures across all files, **STOP**. **Do not** proceed with the rest of Conversion mode.
+2. **Prepare for the task**:
+  - Require the user to explicitly provide output directories for plans, tests, and summaries before writing any files.
+  - Read `references/acceptance-criteria.md`.
+  -  Work one input file at a time. Do not parallelize so that errors in one file's workflow do not affect other files' workflows.
+  -  Derive suite name, test names, startUrl, steps, targets, tags, and source metadata per the rules below.
+3. **JSON test plan**:
+  - Build a JSON test plan that conforms to `references/plan-schema.ts`.
```
Comment (kevanvincent) [inline skills/accelint-ac-to-playwright/references/conversion-mode.md]: This should be 'scripts/plan-schema.ts'

Comment (kevanvincent) [reply skills/accelint-ac-to-playwright/references/conversion-mode.md]: I pushed a quick fix for this and one other section where it pointed to 'references'

**Comment by kevanvincent** [comment]
Overall there are no issues with what you implemented so far - but with the new approach of the conversion and assessment modes I'll need to update the way the eval works to have multi-turn functionality with the SKILL. 

I'll keep this PR open so you can continue with the optimizations once you return

**Comment by tanyafortuna** [comment]
# Testing Plan: AC-to-Playwright Subagent Refactor

## 1. Manual Smoke Testing with Real AC Files

### Goal
Verify critical paths work end-to-end with actual AC files.

### Assessment Mode Tests

**Test 1: PERFECT AC (should pass)**
- Use a known-good AC file with:
  - All targets complete (area.component.intent)
  - All values quoted
  - All verbs recognized
  - All outcomes measurable
- Expected: "✓ AC are conversion-ready"
- Test 1 Result: ✅ PASS

**Test 2: MIXED AC (minor issues only)**
- Create AC with:
  - 2-3 incomplete targets (missing .intent)
  - 1-2 unquoted values
  - Note: I used mixed test files 1 and 4 from the LLM evaluation PR for this.
- Expected:
  - "❌ AC are not conversion-ready"
  - Enumerated issue list with line numbers
  - Specific fixes for each issue
- Test 2 Result: ❌ FAIL - Found an issue with the skill (reload was incorrectly flagged as a precondition and not an action), fixed. 
- Test 2 Result: ✅ PASS - Post-fix, both files passed. The agent doesn't always find the same issues (or all issues) from the test files, but that's an issue for the llm evaluation (fault of the model).

**Test 3: BAD AC (major issues)**
- Create AC with:
  - Vague verbs ("interacts with", "uses")
  - Unmeasurable outcomes ("page updates")
  - Missing targets entirely
  - Note: I used mixed test files 2, 3, and 5 from the LLM evaluation PR for this.
- Expected:
  - "❌ AC are not conversion-ready"
  - Summary of issue types found
  - Offers choice: (a) full question list or (b) interactive mode
- Test 3 Result: ❌ FAIL - Found an issue where the skill classified an unknown action verb as minor rather than major, fixed. 
- Test 3 Result: ⚠️ PARTIAL - The subagent correctly classified AC-B-3.feature as BAD (major issues present) and identified major issues (vague verbs, unmeasurable outcomes), but it's missing the explicit choice offer: "(a) full question list or (b) interactive mode", fixed.
- Test 3 Result: ❌ FAIL - Did not catch a non-When step in the Background section fixed.
- Test 3 Result: ✅ PASS - Post-fixes, all three files passed. The agent doesn't always find the same issues (or all issues) from the test files, but that's an issue for the llm evaluation (fault of the model).

### Conversion Mode Tests

**Test 4: Full pipeline**
- Use simple AC file with 2-3 scenarios
- Expected:
  - Assessment runs first
  - All 9 subagents fire at appropriate times
  - JSON plan validates successfully
  - Test file generates correctly
- Test 4 Result: ❌ FAIL - The agent didn't write files. It seems to have treated the instructions as a description of the workflow rather than instructions it must follow, fixed.
- Test 4 Result: ✅ PASS - sometimes finds false issues or misses issues (model issue), but the workflow works great.

**Test 5: Validation failure**
- Create AC that will fail schema validation
- Expected:
  - diagnose-schema-errors.md subagent spawns
  - Suggests ONE focused fix
  - Retry succeeds after fix
- Test 5 Results:
  - ✅ diagnose-schema-errors subagent spawned correctly
  - ✅ Subagent provided exactly ONE focused fix (testName → name)
  - ✅ Fix was applied successfully - subagent works
  - ❌ FAIL - An error observed: the skill previously made json plans correctly, but now made multiple errors, fixed.
  - ❌ FAIL - An error observed: the skill didn't correctly handle visibility changes, fixed.
  - ❌ FAIL - An error observed: the skill didn't correctly structure targets, fixed.

**Test 6: Translation failure**
- Use AC that causes translation script to fail
- Expected:
  - diagnose-translation-errors.md subagent spawns
  - Diagnoses runtime error
  - Suggests fix
- Test 6 Results:
  - ❌ FAIL - Skill makes up a feature name (suite name) instead of using what's in the file, despite explicit instructions to not infer or make things up.
  - ❌ FAIL - Skill makes up file names and output paths, despite explicit instructions to not infer or make things up.
  - ✅ PASS - successfully found the error and tried to fix it

---

## 2. DELETED

---

## 3. Edge Case Testing

### Goal
Test the new interactive clarification flow variations.

**Test 7: Option (a) - Full question list**
- Trigger BAD AC classification
- Choose option (a)
- Expected: All questions listed at once, grouped by category
- ✅ PASS - formatting was a little wonky but it was all outputted 

**Test 8: Option (b) - Interactive with LLM updates (file path)**
- Trigger BAD AC with file path
- Choose option (b), LLM updates
- Expected:
  - Asks who will update upfront
  - After each answer: updates file, confirms, asks next question
  - After all questions: reassesses automatically
  - If passes: "All set. The AC are now conversion-ready."
- ✅ PASS - worked great

**Test 9: Option (b) - Interactive with LLM updates (pasted AC)**
- Trigger BAD AC with pasted text
- Choose option (b), LLM updates
- Expected:
  - After all questions: reassesses automatically
  - If passes: shares updated AC text with user
- ✅ PASS - it tried to write to a temp file instead of just outputting the updated AC to me in the conversation, which is not my personal preference but still it worked fine 

**Test 10: Option (b) - Interactive with manual updates**
- Trigger BAD AC
- Choose option (b), user updates
- Expected:
  - Just collects answers without updating
  - After all questions: "Update the AC with these changes and I can reassess."
- ✅ PASS - worked great

**Test 11: Reassessment failure loop**
- Trigger BAD AC with LLM updates
- First reassessment still has issues
- Expected: Apologizes, runs assessment mode again with new issue list
- ✅ PASS - worked great

**Test 12: Conversation interruption**
- Start interactive mode with LLM updates
- Answer 2 of 5 questions
- Verify: First 2 updates were applied (if file exists, check it was edited)
- ✅ PASS - worked great

---

## 4. DELETED

---

## 5. DELETED
---

## 6. Integration Testing

### Goal
Test full workflow with realistic scenarios.

**Test 17: Multiple .md files**
- Provide 3 .md files (1 perfect, 1 mixed, 1 bad)
- Expected:
  - All assessed
  - Results reported for all files
  - Does not proceed to conversion
- ✅ PASS - the reporting in the end was a wall of messy text, but it did not proceed to conversion for any of the files (which is correct)

**Test 18: Multiple .feature files**
- Provide 3 .feature files with various issues
- Expected:
  - validate-gherkin-structure.md fires for all
  - Other validators fire as needed
  - Results aggregated correctly
- ❌ FAIL - the results are hideous. They are summarized multiple times because the model will not follow instructions to not "show its thinking". But the results are outputted, even if you have to muddle through them
  - I think this failure is acceptable - getting the results to show more elegantly can come later

**Test 19: Mixed file types**
- Provide mix of .md and .feature files
- Expected:
  - Correct validator chosen based on extension
  - Both formats handled correctly
  - No cross-contamination of validation logic
- ✅ PASS - worked great

**Test 20: Large AC file**
- Use AC with 20+ scenarios
- Note: combined all AC from bad/mixed files into one big file
- Expected:AC
  - All subagents handle large inputs
  - Performance acceptable
  - All issues found (not just first N)
- ✅ PASS - worked great

---

## 7. Specific Bug Scenarios

### Goal
Test scenarios that commonly break in refactors.

**Test 21: Empty/minimal AC**
- Provide AC with only Feature declaration, no scenarios
- Expected: Graceful handling, clear error message
- ✅ PASS - worked great

**Test 22: Special characters in AC**
- Use AC with quotes, apostrophes, special chars in values
- Expected: Proper escaping, no parsing errors
- ✅ PASS - worked great

**Test 23: Incomplete AC file**
- Provide AC that cuts off mid-scenario
- Expected: Structural validation catches it
- ✅ PASS - worked great

**Test 24: Conflicting issues**
- AC with both minor and major issues in same scenario
- Expected: Correctly classified as BAD (major issues take precedence)
- ✅ PASS - worked great

**Comment by kevanvincent** [comment]
@tanyafortuna here are some review comments:

**Real bug**

The new validate-targets.md file tells the subagent to run a target-validator script with npx tsx, importing it from ./skills/accelint-ac-to-playwright/scripts/target-validator.

I had Claude test this. If the subagent is sitting in the skill's own directory (which is exactly what the docs elsewhere in this same PR tell it to do: "cd to the skill directory" before this script runs), the import path doubles up and throws Cannot find module. It only works by accident, when the process happens to be sitting at the root of this specific monorepo. That won't be true for anyone who installs this skill on its own, outside this repo.

There's a second problem stacked on top of it. tsx, the tool used to run the script, isn't actually installed as part of this project. It only shows up as an optional dependency of an unrelated tool (vite). Every other script in this skill runs from a file that's already built and checked in. This is the only one that reaches out to fetch a tool at runtime.

Fix: point the script at the file that's already built (dist/scripts/target-validator.js) instead of the raw source, the same way every other script in the skill does it.

**diagnose-schema-errors.md doesn't match the actual schema**

The action reference table in diagnose-schema-errors.md gets three things wrong against plan-schema.ts. This matters because that file exists specifically to be the authority the diagnostic subagent consults when plan validation fails, so wrong reference data there turns one validation failure into a loop of failures (and the workflow gives up after 2 fix attempts).

1. The table lists the select action as selectOption, but the schema literal is select. If the diagnostic subagent tells the main agent to write "action": "selectOption", the "fix" fails validation again. The confusion probably comes from Playwright's .selectOption() method mentioned in the NEVER-do list, but that's generated test code, not the plan JSON.
2. reload is missing from the table entirely, even though the schema has a reloadStep and validate-actions.md in this same PR treats "reloads" as a valid verb. A diagnostic agent working from this table could flag a legitimate reload step as an unknown action.
3. The table says scroll/wheel direction must be "up" or "down", but the schema allows up, down, left, and right. Notably, validate-actions.md gets this right, so the two subagents contradict each other on the same rule.

Fix: correct the table to use select, add a reload row, and list all four scroll/wheel directions.

**Docs don't match the code**

The CHANGELOG says the new files live in a references/ folder, but they actually live in an agents/ folder. I confirmed this against both the diff and SKILL.md itself, which correctly says agents/.

This matters because anyone reading the CHANGELOG to understand the new file layout gets pointed at paths that don't exist. It looks like the same mistake is fixed inside conversion-mode.md. It just got missed in the CHANGELOG.

Fix: swap references/ for agents/ across the 8 new file paths listed there.

**Small factual slip in the CHANGELOG**

The version note says "Bumped from 1.1.9 to 2.0.0," but the entry listed right above it in the same file is 1.1.10, not 1.1.9.

Low stakes, probably just a copy-paste slip from stacking a new entry on top of an existing one. I also noticed the existing 1.1.10 entry's date quietly changed from 2026-06-11 to 2026-07-09. Don't know if this was intentional or not.

Fix: change "1.1.9" to "1.1.10" in that line.

**Comment by tanyafortuna** [comment]
@kevanvincent Thanks for the feedback you gave. I fixed all four:

1. validate-targets.md now uses node + dist/scripts/target-validator.js instead of npx tsx
2. diagnose-schema-errors.md action table corrected 
3. CHANGELOG paths updated from references/ to agents/
4. CHANGELOG version bump corrected

Ready for re-review.

**Comment by kevanvincent** [comment]
@tanyafortuna everything looks good!

**Comment by tanyafortuna** [comment]
@brandonjpierce @belsrc This PR is ready for review and merge. I tested it (see https://github.com/gohypergiant/agent-skills/pull/120#issuecomment-4985124210 for notes), and Kevan has done a review. 

Can it be merged with the Vercel thing failing? If not, I need some direction on how to get it green.

**Comment by belsrc** [comment]
> @brandonjpierce @belsrc This PR is ready for review and merge. I tested it (see [#120 (comment)](https://github.com/gohypergiant/agent-skills/pull/120#issuecomment-4985124210) for notes), and Kevan has done a review.
> 
> Can it be merged with the Vercel thing failing? If not, I need some direction on how to get it green.

I couldn't say, I don't have access to see what is failing on it (fysa, passed on my PR from yesterday). I think only @shadyendless has access to Vercel?

**Comment by tanyafortuna** [comment]
@brandonjpierce @belsrc checking back in on this post-vacation, any shot at getting it merged? Thanks!

File: skills/accelint-ac-to-playwright/references/acceptance-criteria.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Acceptance criteria guidelines
 
+**When to load this file:** Load when generating JSON plans from AC or running assessment mode. This file contains the authoritative AC writing rules and mappings that inform both assessment and conversion.
```
Comment (brandonjpierce) [inline skills/accelint-ac-to-playwright/references/acceptance-criteria.md:3]: This should go closer to the source file it may be initiated from e.g. SKILL.md. That way an agent knows _before_ reading this file on when it should attempt to read it / follow instructions.

Comment (tanyafortuna) [reply skills/accelint-ac-to-playwright/references/acceptance-criteria.md:3]: Oh, good shout, this is a holdover and I'll remove

File: skills/accelint-ac-to-playwright/agents/assessment-mode.md [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+  - Verify filename is valid:
+    - Contains at least one letter or number
+    - Extension is exactly `.feature` or `.md`
+    - Reject if filename is empty, has no extension, or has invalid extension
+  - Ensure scripts can run:
+    - cd to the skill directory
+    - `npm install`
+    - `npm run build`
+    - This will build the target validator script that will be used later
+2. **Analyze AC text** in two phases:
```
Comment (brandonjpierce) [inline skills/accelint-ac-to-playwright/agents/assessment-mode.md:20]: I would try running `accelint-skill-prose` against these agent files e.g. 

```
/accelint-skill-prose ./skills/accelint-ac-to-playwright/ pay extra close attention to the ./skills/accelint-ac-to-playwright/agents/ files as these act as a skill instruction prompt.
```

One benefit you may see is more favorable serial instruction patterns and patterns around prompting/waiting for a user (where that may be applicable).

File: skills/accelint-ac-to-playwright/agents/assessment-mode.md [hunk 4]
```diff
@@ -0,0 +1,374 @@
...
+  - Once categorized, use the correct template in the Assessment Output Format section to output results
+  - Your FIRST output to the user must be one of the templates from the Assessment Output Format section
+  - Do NOT output anything before the template (no preambles, summaries, or explanations)
+  - The template is the complete output - nothing should come before or after it
+
+## Validation Prompt Templates
+
+### Bullet Format Validation Template
+
+When spawning, use the Agent tool to spawn a subagent with this prompt:
```
Comment (brandonjpierce) [inline skills/accelint-ac-to-playwright/agents/assessment-mode.md:84]: Just a cross harness note, it is not always called `Agent` so you may want to loosen language around this such as:

```
Execute the following prompt with a subagent:
```

We have seen that using phrasing such as "Isolated validator" or similar. E.g. here is a snippet from our `accelint-skill-prose` skill:

> Attempt every required isolated reviewer. A reviewer fails when it cannot start, cannot complete, returns a response that is not valid JSON matching the required Step 2 payload, or returns a required record without source-cited evidence. Do not silently substitute the parent’s own judgment for that reviewer.

File: skills/accelint-ac-to-playwright/references/acceptance-criteria.md (hunk 2 above)
Comment (tanyafortuna) [reply skills/accelint-ac-to-playwright/references/acceptance-criteria.md]: Removed

File: skills/accelint-ac-to-playwright/agents/assessment-mode.md (hunk 4 above)
Comment (tanyafortuna) [reply skills/accelint-ac-to-playwright/agents/assessment-mode.md]: addressed

File: skills/accelint-ac-to-playwright/agents/assessment-mode.md (hunk 3 above)
Comment (tanyafortuna) [reply skills/accelint-ac-to-playwright/agents/assessment-mode.md]: I ran the skill prose agent and addressed the three feedbacks it had. Pasting in the report for posterity (not that I expect anyone to read it)
 
 ----
 
 # Skill Prose Audit: accelint-ac-to-playwright

## Review Coverage

**Incomplete rubric coverage:** Two required reviewers could not complete due to missing rubric files in the target skill directory:

- **Serial instruction and workflow integrity**: `references/serial-instruction-guidance.md` not found
- **User-question and waiting behavior**: `references/user-question-waiting.md` not found

The rubric files exist in the accelint-skill-prose skill but not in the target accelint-ac-to-playwright skill. These reviews would need the rubric files copied to the target skill or the review scope adjusted.

**Completed reviews:**
- Normative-language and obligation precision ✓
- STE-compatible clarity and usability ✓

---

## Category-Level Audit

### 1. Normative-Language and Obligation Precision
**Grade:** 0 (Critical behavioral defect)  
**State:** Finding  
**Priority:** Medium

**Issue:** Agent obligations use lowercase "must" instead of RFC 2119 MUST, softening mandatory workflow requirements. An LLM may interpret "you must execute" as advice rather than requirement.

**Evidence:**
- `agents/assessment-mode.md` lines 5-7: "You must spawn the validation subagents"
- `agents/assessment-mode.md` line 24: "You must complete both phases"
- `agents/conversion-mode.md` lines 5, 43, 186: Multiple lowercase "must" for mandatory agent behavior

**Risk:** Agents may skip critical workflow steps (spawning validation subagents, running assessment before conversion) if they interpret lowercase "must" as advisory.

**Change classification:** Wording-only

---

### 2. STE-Compatible Clarity and Usability
**Grade:** 2 (Needs revision)  
**State:** Finding  
**Priority:** High

**Issue:** Instruction and explanation mixed throughout, creating ambiguity about what is operational directive vs rationale.

**Evidence:**
- `agents/assessment-mode.md` lines 5-8: Rationale ("that you don't have loaded", "even if you're in plan mode") embedded in instructions
- `agents/assessment-mode.md` lines 370-375: NEVER prohibitions mixed with explanatory phrases ("for context-management purposes", "you are wasting tokens")
- `agents/conversion-mode.md` lines 118-123: Open-ended phrase "similar wording" introduces qualitative judgment without controlled vocabulary anchor
- `agents/conversion-mode.md` lines 170-186: Extensive rationale embedded within NEVER prohibitions instead of separated

**Risk:** Agents may treat rationale as additional conditions or ignore prohibitions as explanatory context. Qualitative gates create unbounded judgment calls.

**Change classification:** Behavior-preserving structural rewrite

---

## Prioritized Rewrite Proposal

### Change 1: Capitalize agent obligation "must" → "MUST"
**Addresses:** Normative-language finding  
**Target:** 7 occurrences across assessment-mode.md and conversion-mode.md  
**Behavior preservation:** All workflow gates and mandatory steps remain unchanged; only capitalization changes for RFC 2119 compliance  
**Classification:** Wording-only

**Specific changes:**
- `assessment-mode.md` line 5: "You must spawn" → "You MUST spawn"
- `assessment-mode.md` line 7: "You must execute" → "You MUST execute"  
- `assessment-mode.md` line 24: "You must complete" → "You MUST complete"
- `conversion-mode.md` line 5: "You must execute" → "You MUST execute"
- `conversion-mode.md` line 43: "you must stop" → "you MUST stop"
- `conversion-mode.md` line 186: "you must always run" → "you MUST always run"

---

### Change 2: Separate rationale from instructions
**Addresses:** STE-compatible clarity finding  
**Target:** NEVER statement sections and workflow instructions  
**Behavior preservation:** All prohibitions, gates, and workflow sequencing unchanged; rationale moves to dedicated subsections  
**Classification:** Behavior-preserving structural rewrite

**Structural changes:**

**In `agents/assessment-mode.md` lines 368-375:**

Current structure mixes prohibition and rationale:
```markdown
- **NEVER read any files other than `acceptance-criteria.md`, `test-hooks.md`, and any user-provided AC files** - for context-management purposes, you spawn subagents to efficiently do tasks. If you read these files you are wasting tokens.
```

Proposed structure separates them:
```markdown
- **NEVER read any files other than `acceptance-criteria.md`, `test-hooks.md`, and any user-provided AC files**

**Why this matters:** For context-management purposes, you spawn subagents to efficiently do tasks. Reading these files wastes tokens.
```

Apply same pattern to all NEVER statements with embedded rationale.

**In `agents/conversion-mode.md` lines 170-186:**

Move extensive rationale ("breaks Playwright's navigation lifecycle and causes race conditions...") to dedicated "Why this matters" subsections after each prohibition.

---

### Change 3: Replace qualitative gate with explicit reference
**Addresses:** STE-compatible clarity finding  
**Target:** `agents/conversion-mode.md` line 118  
**Behavior preservation:** Visibility assertion requirements unchanged; gate becomes explicit  
**Classification:** Wording-only

**Current:** "For visibility changes (e.g., visible/appears/shows/hides and similar wording)"

**Proposed:** "For visibility changes (words: visible, appears, shows, hides, disappears, see, seen)"

This removes open-ended "similar wording" qualifier with explicit enumeration.

---

## Critical Files to Modify

1. `skills/accelint-ac-to-playwright/agents/assessment-mode.md` (lines 5, 7, 24, 368-375)
2. `skills/accelint-ac-to-playwright/agents/conversion-mode.md` (lines 5, 43, 118, 170-186)

---

## Verification

After changes:
1. Grep for remaining lowercase "you must" in agent files
2. Verify NEVER statements have separated rationale sections
3. Verify "similar wording" replaced with explicit list
4. Run skill in test mode to confirm workflow behavior unchanged

**Comment by tanyafortuna** [comment]
@brandonjpierce I addressed your three code review comments and this is ready for re-review!
